### ISSUE TITLE
Cannot show bbcukmet weather data since Oct 1 2020

### ISSUE THREAD

**Issue by thegreyshadow** [Zren/plasma-applet-condensedweather#5]
Thanks for developing this Plasmoid.

I cannot see any BBC Weather data since October 1, 2020. Looks like the API or something upstream was changed.

**Comment by Zren** [comment]
"City of London, Greater London" works for me. Does it work for you?

![2020-10-02___14-46-05](https://user-images.githubusercontent.com/416367/94958713-0179f800-04be-11eb-987f-fe6ee95794fb.png)

Are you sombragris from https://store.kde.org?

![2020-10-02___14-47-58](https://user-images.githubusercontent.com/416367/94958888-456cfd00-04be-11eb-8fe2-3ebd41197996.png)

**Comment by thegreyshadow** [comment]
1. I'm in Asunción, Paraguay, South America.
This is why it's important to have bbcukmet available. It is (was?) the only service with good info available. The other source reporting on my country among the sources (wetter.com) is not so complete.

2. Yes, I'm "sombragris" on store.kde.org :)

If I can help debugging you this problem I'd be glad. Thanks!

**Comment by Zren** [comment]
https://www.metoffice.gov.uk/weather/forecast/6ex0282dy#?date=2020-10-02

Seems to still have data on their website.

* https://invent.kde.org/plasma/plasma-workspace/-/tree/master/dataengines/weather/ions/bbcukmet
* https://invent.kde.org/plasma/plasma-workspace/-/blob/master/dataengines/weather/ions/bbcukmet/ion_bbcukmet.cpp
* http://www.bbc.com/locator/default/en-GB/search.json?search=Asuncion&filter=international&postcode_unit=false&postcode_district=true

Searching for `Asuncion` still finds the station.


id | "3439389"
-- | --
name | "Asunción"
fullName | "Asunción, Paraguay"

* https://weather-broker-cdn.api.bbci.co.uk/en/forecast/rss/3day/3439389

Nothing? Just displays a JSON HTTP 202 response.

Lets try london.


id | "2643743"
-- | --
name | "London"
fullName | "London, Greater London"


* https://weather-broker-cdn.api.bbci.co.uk/en/forecast/rss/3day/2643743

Which downloads an RSS file instead. Weird.

TL;DR: The website no longer supports your station. It's not a bug in this widget or the bbkcomet weather ion code. It could be a temporary lack of data or bug on the websites side, so I'd check it again in a week.

**Comment by thegreyshadow** [comment]
Thanks for this effort. It's weird as you say.

I understand it's not a bug in the widget code butt, as I said, it was some change upstream.

I suspect my location is not the only one affected.

I think the report may be closed since your code works as intended, but perhaps you might consider adding another weather source such as openweathermap.org. In any case, I'm really grateful.

**Comment by thegreyshadow** [comment]
Today I get data from the BBC as usual. This was obviously a fully upstream issue so I think it's best to close the report. Thanks for helping me sorting out.
